tests/InputFiles/ScalarWave/SphericalWave3DPeriodic.yaml

```diff
@@ -0,0 +1,54 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# Executable: EvolveRegularSphericalWave3D
+# Check: parse;execute
+
+AnalyticSolution:
+  RegularSphericalWave:
+    Profile:
+      Gaussian:
+        Amplitude: 1.0
+        Width: 1.0
+        Center: 0.0
+
+Evolution:
+  InitialTime: 0.0
+  InitialTimeStep: 0.001
+  InitialSlabSize: 0.01
+  TimeStepper:
+    AdamsBashforthN:
+      Order: 3
+  StepController: BinaryFraction
+  StepChoosers:
+    - Constant: 0.05
+    - Increase:
+        Factor: 2
+    - Cfl:
+        SafetyFactor: 0.2
+
+DomainCreator:
+  Brick:
+    LowerBound: [0.0, 0.0, 0.0]
+    UpperBound: [6.0, 6.0, 6.0]
+    IsPeriodicIn: [true, true, true]
+    InitialRefinement: [1, 1, 1]
+    InitialGridPoints: [5, 5, 5]
+
+# If filtering is enabled in the executable the filter can be controlled using:
+# Filtering:
+#   ExpFilter0:
+#     Alpha: 12
+#     HalfPower: 32
```

Comment:
You can remove this commented-out section, it’s not being tested

Reply:
While it is not being tested, I would strongly prefer to keep this block in for easy reference when one wants to use filtering, which is anyway compiled for a 2D system. I believe the purpose of input files is not just to test executables, but also to provide easy reference to users who might want to evolve a system.